frontend/src/components/editor-page/sidebar/specific-sidebar-entries/aliases-sidebar-entry/aliases-modal/aliases-list-entry.tsx

```diff
@@ -45,35 +46,32 @@ export const AliasesListEntry: React.FC<AliasesListEntryProps> = ({ alias }) =>
   const isPrimaryText = useTranslatedText('editor.modal.aliases.isPrimary')
   const makePrimaryText = useTranslatedText('editor.modal.aliases.makePrimary')
   const removeAliasText = useTranslatedText('editor.modal.aliases.removeAlias')
+  const primaryLabel = useTranslatedText('editor.modal.aliases.primaryLabel')
```

Comment:
The `useTranslatedText` hook is only used when we need translated texts in props. In this case the text is used as a children for the Badge. Please use the `<Trans>` component in such cases.

Reply:
Thank you for your instruction 😄 
I fixed it!

Reply:
@mrdrogdrog I would be happy if you could review it when you have time 😄